pkg/tfgen/installation_docs_test.go

```diff
@@ -157,6 +157,57 @@ func TestWriteInstallationInstructions(t *testing.T) {
 	})
 }
 
+func TestWriteOverviewHeader(t *testing.T) {
```

Comment:
I'd love if this was a bit higher level, probably by the scope of one function. We shouldn't need to add the header to the body in the test content.

Reply:
The next level up is testing the whole file conversion. I will add this in another pull request today.
Adding the header to the body here provides more surrounding context, which I was hoping would provide more clarity.